src/web-auth/index.js

```diff
@@ -283,6 +283,14 @@ WebAuth.prototype.validateAuthenticationResponse = function(options, parsedHash,
     if (validationError.error !== 'invalid_token') {
       return callback(validationError);
     }
+    if (!parsedHash.access_token) {
+      var noAccessTokenError = {
+        error: 'invalid_token',
+        description: 'The id_token cannot be validated because it was signed with the HS256 algorithm and public clients (like a browser) can’t store secrets. Please read the associated doc for possible ways to fix this.',
+        error_uri: 'https://auth0.com/docs/errors/libraries/auth0-js/invalid-token#parsing-an-hs256-signed-id-token-without-an-access-token'
```

Comment:
new

Reply:
Why are you hardcoding the URL? IMO this is not easy to maintain. Anyway, instead of adding a new property to the error object I'd append it to the description.